modules/products/idea/src/main/kotlin/com/metalbear/mirrord/products/idea/IdeaRunConfigurationExtension.kt

```diff
@@ -63,9 +63,8 @@ class IdeaRunConfigurationExtension : RunConfigurationExtension() {
             else -> null
         }
 
-        val mirrordEnv = service.execManager.wrapper("idea").apply {
+        val mirrordEnv = service.execManager.wrapper("idea", null).apply {
```

Comment:
Nooo, not null. Either `params.env` or `(configuration as ExternalSystemRunConfiguration).settings.env`. See `getMirrordConfigPath` above

Reply:
Changed this to:

```kotlin
val extraEnv = params.env + (configuration as ExternalSystemRunConfiguration).settings.env
```

Is this the right thing?